### ISSUE TITLE
TrackedReferencesRepository uses Mapper that is not in the included DLLs

### ISSUE THREAD

**Issue by LennardF1989** [umbraco/Umbraco-CMS#14150]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v11.2.2

### Bug summary

Also applies to the latest v11.3.0.

See these line: 
- https://github.com/umbraco/Umbraco-CMS/blob/33adbf41fa1f5c5d0759c70a7116114107addf56/src/Umbraco.Infrastructure/Persistence/Repositories/Implement/TrackedReferencesRepository.cs#L304
- https://github.com/umbraco/Umbraco-CMS/blob/33adbf41fa1f5c5d0759c70a7116114107addf56/src/Umbraco.Infrastructure/Persistence/Repositories/Implement/TrackedReferencesRepository.cs#L356
- https://github.com/umbraco/Umbraco-CMS/blob/33adbf41fa1f5c5d0759c70a7116114107addf56/src/Umbraco.Infrastructure/Persistence/Repositories/Implement/TrackedReferencesRepository.cs#L419

These use the UmbracoMapper to do their thing, the other methods use `MapDtoToEntity`.

The mapper used, however, is not actually part of the current CMS but the new one: https://github.com/umbraco/Umbraco-CMS/blob/33adbf41fa1f5c5d0759c70a7116114107addf56/src/Umbraco.New.Cms.Infrastructure/Persistence/Mappers/RelationModelMapDefinition.cs

A call to any of these methods using this mapper will fail! Took quite a bit to figure out... since from a code perspective the mapper is there. The sneaky part is that it's in a DLL that's not part of the NuGet packages.

The fix would be to move this mapper to the current Infrastructure code-base.

**Comment by Zeegaan** [comment]
I can definitely reproduce this 🙈

Here is the sample code I used to trigger this:

```
using Umbraco.Cms.Core.Composing;
using Umbraco.Cms.Core.Events;
using Umbraco.Cms.Core.Notifications;
using Umbraco.Cms.Core.Persistence.Repositories;
using Umbraco.Cms.Infrastructure.Scoping;

namespace TestTrack;

public class MyNotiHandler : INotificationHandler<UmbracoApplicationStartedNotification>
{
    private readonly ITrackedReferencesRepository _trackedReferencesRepository;
    private readonly IScopeProvider _scopeProvider;

    public MyNotiHandler(ITrackedReferencesRepository trackedReferencesRepository, IScopeProvider scopeProvider)
    {
        _trackedReferencesRepository = trackedReferencesRepository;
        _scopeProvider = scopeProvider;
    }
    public void Handle(UmbracoApplicationStartedNotification notification)
    {
        using var scope = _scopeProvider.CreateScope();
        long skip = 0;
        long take = 10;
        var result = _trackedReferencesRepository.GetPagedRelationsForItem(1057, skip, take, false, out long totalRecords);

        scope.Complete();
    }
}

public class MyClass : IComposer
{
    public void Compose(IUmbracoBuilder builder)
    {
        builder.AddNotificationHandler<UmbracoApplicationStartedNotification, MyNotiHandler>();
    }
}
```

**Comment by elit0451** [comment]
Fixed in #14153
